integration_tests/src/main/python/csv_test.py

```diff
@@ -463,14 +463,15 @@ def test_input_meta_fallback(spark_tmp_path, v1_enabled_list, disable_conf):
     updated_conf = copy_and_update(_enable_all_types_conf, {
         'spark.sql.sources.useV1SourceList': v1_enabled_list,
         disable_conf: 'false'})
-    assert_gpu_and_cpu_are_equal_collect(
+    assert_gpu_fallback_collect(
             lambda spark : spark.read.schema(gen.data_type)\
                     .csv(data_path)\
                     .filter(f.col('a') > 0)\
                     .selectExpr('a',
                         'input_file_name()',
                         'input_file_block_start()',
                         'input_file_block_length()'),
+            cpu_fallback_class_name = 'FileSourceScanExec' if v1_enabled_list == 'csv' else 'BatchScanExec',
```

Comment:
not a must-fix, stylistically and because in a future the parameter `v1_enabled_list` might become a comma-separated value or a Python list we might want to check for  `'csv' in v1_enabled_list` rather than equality.